mmf/utils/checkpoint.py

```diff
@@ -582,5 +585,39 @@ def restore(self):
 
     def finalize(self):
         if is_master() or is_xla():
-            with PathManager.open(self.pth_filepath, "wb") as f:
+            with open_if_master(self.pth_filepath, "wb") as f:
                 self.save_func(self.trainer.model.state_dict(), f)
+
+
+def save_xla_ckpt(ckpt, file_or_path):
```

Comment:
This is awkwardly placed. Move to top or some other utils files such `xla.py` (create if not there).